CHANGELOG.md

```diff
@@ -1,5 +1,87 @@
 # CHANGELOG
 
+## Unreleased
+
+### BREAKING CHANGES
+
+- The `TMHOME` environment variable was renamed to `CMTHOME`, and all environment variables starting with `TM_` are instead prefixed with `CMT_`
+  ([\#211](https://github.com/cometbft/cometbft/issues/211))
+- `[protobuf]` Remove fields `sender`, `priority`, and `mempool_error` from
+  `ResponseCheckTx`. ([\#260](https://github.com/cometbft/cometbft/issues/260))
+- `[mempool]` Remove priority mempool.
+  ([\#260](https://github.com/cometbft/cometbft/issues/260))
+- `[config]` Remove `Version` field from `MempoolConfig`.
+  ([\#260](https://github.com/cometbft/cometbft/issues/260))
+- Bump minimum Go version to 1.20
+  ([\#385](https://github.com/cometbft/cometbft/issues/385))
+- `[tools/tm-signer-harness]` Set OS home dir to instead of the hardcoded PATH.
```

Comment:
Is there something missing in this description?

Reply:
Didn't we remove tm-signer-harness though? If so we can actually delete this changelog entry.

Reply:
Yeah, deleting it should do. My comment was that there seems to be something missing in the sentence

Reply:
Seems like that was the title of the original PR. The tm-signer-harness used to use a hard-coded path, and the PR changed it to use the Go std lib functionality to obtain the user's home dir instead.

Reply:
I see. So it's the "to" preposition that go me confused... "Set OS home dir instead of the hardcoded PATH" makes sense to me